product/quickstart/component-basics.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: "Component basics"
+description: "Learn how to use and customize components in Makeswift."
+icon: "cube" iconType: "solid"
+---
+
+Spend a few seconds exploring the Canvas and its components. You can click on any component in the Canvas to see its corresponding properties in the Properties Sidebar.
+
+<Frame>
+  <video
+    autoPlay
+    muted
+    loop
+    playsInline
+    title="Components and properties"
+    className="w-full aspect-video"
+    src="/images/quickstart/components-and-properties.mp4"
+  ></video>
+</Frame>
```

Comment:
That's a good call. Any suggestions on how to differentiate? I want to be zoomed in and I thought it would be challenging to zoom to a perfect height that made sense to only show some properties in the middle?